core/controllers/editor_test.py

```diff
@@ -25,9 +26,12 @@
 from core.domain import stats_domain
 from core.domain import rights_manager
 from core.domain import rule_domain
+from core.platform import models
 from core.tests import test_utils
 import feconf
 
+(user_models,) = models.Registry.import_models([models.NAMES.user])
```

Comment:
@seanlip Sorry my bad. The get method is used in a couple of test bodies.
exp_user_data = user_models.ExplorationUserDataModel.get_by_id(). However If I don't use it, what are my alternatives ?